src/Features/CSharp/Portable/ConvertToRecord/PositionalParameterInfo.cs

```diff
@@ -5,39 +5,78 @@
 using System.Collections.Immutable;
 using System.Linq;
 using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.ConvertToRecord
+namespace Microsoft.CodeAnalysis.CSharp.ConvertToRecord
 {
-    internal record PositionalParameterInfo(PropertyDeclarationSyntax Declaration, IPropertySymbol Symbol, bool KeepAsOverride)
+    /// <summary>
+    /// Represents a property that should be added as a positional parameter
+    /// </summary>
+    /// <param name="Declaration">Original declaration, null iff IsInherited is true
```

Comment:
If the "if and only if" here is true, do we actually need both members? Or can we just make IsInherited => Declaration != null to prevent any possible mismatches?